CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Addresses in ConnectionPool may be changed from an external code (#208)
 - ConnectionPool recreates connections too often (#208)
 - A connection is still opened after ConnectionPool.Close() (#208)
+- Future.GetTyped() after Future.Get() does not decode response
```

Comment:
So is it a bugfix or feature introduction? Commit message seems more like a second one.

Reply:
Thank you. Yes, it can be confusing. I've updated the commit message.